Documentation/release-notes/debugtype.md

```diff
@@ -0,0 +1,21 @@
+#### Deprecation of DebugType full and pdbonly
+
+* warning XA0125: 'AndroidApp1.pdb' is using a deprecated debug
+  information level. Set the debugging information to Portable in the
+  Visual Studio project property pages or edit the project file in a
+  text editor and set the 'DebugType' MSBuild property to 'portable'
+  to use the newer, cross-platform debug information level.
```

Comment:
As a tiny formatting idea to match some similar previous release notes, this could use a code block for the example warning:

````markdown
```
warning XA0125: 'AndroidApp1.pdb' is using a deprecated debug information level. Set the debugging information to Portable in the Visual Studio project property pages or edit the project file in a text editor and set the 'DebugType' MSBuild property to 'portable' to use the newer, cross-platform debug information level.
```
````